### PR TITLE
VK Bridge Game Events

### PR DESCRIPTION
**Pull request by Evgenii-Zinner**
Two functions from VK Bridge for games was added, 
VKWebAppShowInviteBox Send an Invitation;
VKWebAppShowLeaderBoardBox Open the Leaderboard.

### CODE REVIEW

File: lib/src/bridge/vk_bridge.dart [hunk 1]
```diff
@@ -184,6 +184,18 @@ abstract class VKBridge {
...
+  /// [VKWebAppShowInviteBox] invokes a request window
+  /// for inviting friends to the game.
+  ///
+  /// Platforms: iOS, Android, Web, Mobile Web
+  Future<VKWebAppBoolResult> showInviteBox();
+
+  /// [VKWebAppShowLeaderBoardBox] invokes a request window
+  /// for inviting friends to the game.
+  ///
+  /// Platforms: iOS, Android, Web, Mobile Web
```
Comment (sanekyy) [inline lib/src/bridge/vk_bridge.dart:196]: There is no Web support or there is an error in official documentation.

https://dev.vk.com/bridge/VKWebAppShowLeaderBoardBox

File: lib/src/bridge/vk_bridge.dart [hunk 2]
```diff
@@ -184,6 +184,18 @@ abstract class VKBridge {
...
+  Future<VKWebAppBoolResult> showLeaderBoardBox();
```
Comment (sanekyy) [inline lib/src/bridge/vk_bridge.dart:197]: As I see there is a optional parameter `user_result`

https://dev.vk.com/bridge/VKWebAppShowLeaderBoardBox

File: lib/src/bridge/vk_bridge.dart [hunk 3]
```diff
@@ -184,6 +184,18 @@ abstract class VKBridge {
   /// Platforms: iOS, Android, Web, Mobile Web
   Future<VKWebAppBoolResult> addToFavorites();
 
+  /// [VKWebAppShowInviteBox] invokes a request window
+  /// for inviting friends to the game.
+  ///
+  /// Platforms: iOS, Android, Web, Mobile Web
+  Future<VKWebAppBoolResult> showInviteBox();
```
Comment (sanekyy) [inline lib/src/bridge/vk_bridge.dart:191]: Result contains `notSentIds`

https://dev.vk.com/bridge/VKWebAppShowInviteBox

Comment (sanekyy) [inline lib/src/bridge/vk_bridge.dart:191]: As I see there is a optional parameter `requestKey`

**Review by sanekyy — CHANGES_REQUESTED** [review]

**Comment by otopba** [comment]
@Evgenii-Zinner Hello! Is this PR still relevant?

**Pull request closed without merging**